src/runtime/types.ts

```diff
@@ -25,19 +25,24 @@ export interface IStyleColor {
 }
 
 export interface IStyleFont {
+  lineHeight?: string;
   cssFontFamily?: string;
-  family?: string;
+  family?: string | string[];
   index?: number;
   fontStyleParam?: boolean;
   fontParam?: boolean;
   preset?: string;
-  size?: number;
-  style?: {
-    bold: boolean;
-    italic: boolean;
-    underline: boolean;
-  };
-  value: string;
+  size?: number | string;
+  weight?: string;
+  variant?: string;
+  style?:
+    | {
+        bold: boolean;
+        italic: boolean;
+        underline: boolean;
+      }
+    | string;
+  value?: string;
```

Comment:
found in tests that such shape is available (otherwise need to investigate, what real-world usage of this and change tests according to previous type).